### PR TITLE
RATEST-135: Fix Add User Test

### PR DESCRIPTION
**Pull request by kdaud**
Ticket ID: https://issues.openmrs.org/browse/RATEST-135

Description
Fix add user test

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     private static final String URL = "/admin/index.htm";
 
-    private static final By MANAGE_USERS = By.cssSelector("#legacyui-manageUsers a");
```
Comment (sherrif10) [inline ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: Any reason why changing to the defined full url, i think this should be left as a tag not defined content url

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: > Any reason why changing to the defined full url

Investigation report indicates clearly that the current property attribute for the [element](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/285dcc90731099bd2d9368097428a21b3c8bcca2/ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java#L19) does not exist. Apart from the suggested property, the element can also be retrieved via its `id, xpath, linkText and others` but in terms of` elementLoadTime`, the suggested one takes less time to retrieve the element than any other attribute can do. Putting quality assurance in mind, the suggested property is given a priority

Comment (sherrif10) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: > Investigation Lab report indicates clearly that the current property attribute for the [element](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/285dcc90731099bd2d9368097428a21b3c8bcca2/ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java#L19) does not exist.

The element does not exist ??

Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java:19]: > The element does not exist ??

Lab report indicates clearly that the current property attribute for the [element](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/285dcc90731099bd2d9368097428a21b3c8bcca2/ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java#L19)

**Pull request closed without merging**